contracts/src/IncredibleSquaringTaskManager.sol

```diff
@@ -73,25 +73,25 @@ contract IncredibleSquaringTaskManager is
     constructor(
         ISlashingRegistryCoordinator _registryCoordinator,
         IPauserRegistry _pauserRegistry,
-        uint32 _taskResponseWindowBlock,
-        address _serviceManager
+        uint32 _taskResponseWindowBlock
     ) BLSSignatureChecker(_registryCoordinator) Pausable(_pauserRegistry) {
         TASK_RESPONSE_WINDOW_BLOCK = _taskResponseWindowBlock;
-        serviceManager = _serviceManager;
     }
 
     function initialize(
         address initialOwner,
         address _aggregator,
         address _generator,
         address _allocationManager,
-        address _slasher
+        address _slasher,
+        address _serviceManager
```

Comment:
Why this change?

Reply:
when we put it in constructor , serviceManager is coming  address(0) .